src/Playwright/Transport/Connection.cs

```diff
@@ -276,8 +272,7 @@ internal void Dispatch(PlaywrightServerMessage message)
         {
             if (message.Method == "__create__")
             {
-                var createObjectInfo = message.Params.Value.ToObject<CreateObjectInfo>(DefaultJsonSerializerOptions);
-                CreateRemoteObject(message.Guid, createObjectInfo.Type, createObjectInfo.Guid, createObjectInfo.Initializer);
+                CreateRemoteObject(message.Guid, message.Params.Value.GetProperty("type").ToObject<ChannelOwnerType>(), message.Params.GetString("guid", false)!, message.Params?.GetProperty("initializer"));
```

Comment:
Same, the previous code was arguably more type-safe.

Reply:
Internally its similar and more explicit without going through the `JsonSerializer` infra + we don't need this extra type. I think for use-cases where we don't need to share a type this is preferred (like we do in other places as well).